docs/design/DB/dokuha-roadmap.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Users (
+	user_id SERIAL PRIMARY KEY,
+	username VARCHAR(255) NOT NULL,
+	password VARCHAR(255) NOT NULL
```

Comment:
Users にも作成日と更新日を入れていいと思う〜！
リリースした後に何か起きたときの調査などでも使えるし、多分ユーザー名の編集とかはできるようにすると思うので

逆にいらないと思った理由があったら後学のために話したいです！

```suggestion
	password VARCHAR(255) NOT NULL,
	created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
	updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```